t/27-consoles-s3270.t

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/perl
 
+use Test::Mock::Time;
```

Comment:
Did you put it into the first line on purpose? If possible move it below the Test::Most

Reply:
Ok Sure

Reply:
Sorry, actually it should also be below the next line to ensure that we use the strict and signature support from Mojo::Base unless that makes problems.

Reply:
Originally, I thought if I add `use Test::Mock::Time;` at the top, other modules coming after this can also utilize this mock if they use sleep or time functionality.
I will move it below `use Mojo::Base -strict, -signatures;`